implementations/elixir/ockam/ockly/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Ockly.MixProject do
   def project do
     [
       app: :ockly,
-      version: "0.1.0",
```

Comment:
We now set the version of Ockly to that of the Ockam rust crate, this is so that we can download a defined version of our NIF, @polvorin should we automate updating this version during release?

Reply:
if we don't set `OCKAM_DOWNLOAD_NIF` it will build it from the current checkout regardless right? (that,  compile from the current source is what we would want to do almost always,  in CI and developer' machines)

Reply:
I currently set the version of Ockly (elixir) to version 0.117.0, do we want to automatically update the version for every release (so that we can download latest NIFs), or we update manually?